docs/reference/ops-tracing.rst

```diff
@@ -8,12 +8,24 @@
 Open Telemetry resource attributes
 ----------------------------------
 
+The following well-known and additional resource attributes are set:
```

Comment:
Is "well-known" here meaning it's like a defacto standard, almost everyone sets these? Or is this like the well-known URI where it's actually part of a strict definition somewhere?

If there's a link that can be provided that explains that (some of) these are typically set, like you have for the Juju topology ones, that would be really handy I think.

Right now, it seems like there's considerable overlap between the `service.` ones and the Juju topology ones. I'm guessing that there's a valid reason for that -- basically that the `service.` ones are expected by something, and that Canonical has standardised on the Juju topology ones -- but it'd be nice to be clear on the reason for the duplication/overlap, which I think a link would provide.

Reply:
Well-known as in the attribute keys are suggested/defined upstream, by OpenTelemetry.

Reply:
I've update the text, PTAL.